internal/v4/stats.go

```diff
@@ -122,7 +122,7 @@ func (h *ReqHandler) serveStatsUpdate(_ http.Header, r *http.Request) (interface
 		return nil, errgo.WithCausef(nil, params.ErrMethodNotAllowed, "%s not allowed", r.Method)
 	}
 
-	var req params.StatsUpdateRequest
+	var req []params.StatsUpdateRequest
```

Comment:
Rather than have a slice here, we've been quite careful elsewhere to use
only a struct as an API parameter because that is more future-proof (it's easy
to add fields and maintain backward compatiblity). I suggest we change StatsUpdateRequest to be a struct holding a slice member.

That means that if we want to make things more efficient in the future (for example by passing a map[url][]timestamp) then that possibility remains open.